<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dicom_Series_Reader</class>
 <widget class="QMainWindow" name="Dicom_Series_Reader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dicom_Series_Reader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="myQVTKWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>681</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="mySliderSlices">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_max_slice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_min_slice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFile_2"/>
   </widget>
   <widget class="QMenu" name="menuFolder">
    <property name="title">
     <string>Folder</string>
    </property>
    <addaction name="actionFolder_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFolder"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="actionFolder_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFile_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
